css_task2/style.css

```diff
@@ -0,0 +1,64 @@
+.container{
+    display:flex;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 14px;
+}
+main{
+    width:50%;
+}
+.buttons{
+    float: right;
```

Comment:
Please don't mix flex with float. You should use 
    display: flex;
    justify-content: flex-end;
    align-items: center;